offline/cable_cru_TRENDY.F90

```diff
@@ -408,38 +551,80 @@ SUBROUTINE cru_get_daily_met(CRU, LastDayOfYear)
   ! opening and closing the io stream unnecessarily, but I think it's a minor
   ! evil.
 
-  ! Address prevTmax first
+  ! There's a serious (?) question to be answered here- what do we do about leap
+  ! years? There are two ideas which are at odds with eachother:
+  ! 1. We want to retrieve that correct data for the relevant year used for
+  !   recycling, which implies leap years should be based on the met recycling
+  !   year. This prevents instances where we try to access the 366th day of a
+  !   year with 365 days, or only get 365 days from a 366 day year.
+  ! 2. We often want to perform yearly averages or accumulations over a year.
+  !   These accumulations/averages occur elsewhere in the code, and thus use the
+  !   number of days from the core simulation year. This imples leap years
+  !   should be based on the core simulation year, to prevent instances where
+  !   we accumulate meteorology over 366 days but average over 365 days and vice
+  !   versa.
```

Comment:
I think this may come down to a question of exactly how much flexibility do we want with the forcing- in the original TRENDY set-up, there were specific ```RunType```s which recycled some variables but used the simulated year for others, which is why I included the tailored recycling options. I think there are instances where this tailoring could be useful, in instances where some variables have a longer history than others. But it would make the handling easier if we just set a blanket "Met recycling on or off".